playstore/validator.go

```diff
@@ -34,7 +34,7 @@ type IABSubscription interface {
 	CancelSubscription(context.Context, string, string, string) error
 	RefundSubscription(context.Context, string, string, string) error
 	RevokeSubscription(context.Context, string, string, string) error
-	DeferSubscription(context.Context, string, string, string, *androidpublisher.SubscriptionPurchasesDeferRequest) error
+	DeferSubscription(context.Context, string, string, string, *androidpublisher.SubscriptionPurchasesDeferRequest) (*androidpublisher.SubscriptionPurchasesDeferResponse, error)
```

Comment:
Aligning this method with https://github.com/awa/go-iap/blob/784c6b7c8e532634aca9bebfa1c32b793743b0a4/playstore/validator.go#L227-L228